### PR TITLE
National assembly constituencies

### PR DESCRIPTION
**Pull request by GeoWill**

### CODE REVIEW

**Comment by GeoWill** [comment]
NB Q12857471 appears in the build warnings, but is a former constituency. Haven't tracked down what pulls it in.

**Comment by alexdutton** [comment]
> NB Q12857471 appears in the build warnings, but is a former constituency. Haven't tracked down what pulls it in.

There was an incorrect district on [this membership](https://www.wikidata.org/wiki/Q4790529#Q4790529$04d1c1bc-429c-b428-10bc-3b66d10cb4f0), which I've now corrected (and end-dated to, as he's now president). I'll refresh this legislature in a commit on this branch.

**Review by alexdutton — APPROVED** [review]
I'm happy with this now, with the re-parenting in the fixup commits. Can you check you agree and then either squash and merge, or tell me to do it? :pray:

File: boundaries/build/index.json [hunk 1]
```diff
@@ -12,6 +12,18 @@
       "lang:en": "NAME_EN"
     }
   },
+  {
+    "directory": "flacs",
+    "area_type_wikidata_item_id": "Q1062177",
+    "associations": [],
```
Comment (GeoWill) [inline boundaries/build/index.json]: Should Q28172635 be an association in here?

**Comment by GeoWill** [comment]
One comment in line, otherwise looks good.

File: boundaries/build/index.json (hunk 1 above)
Comment (alexdutton) [reply boundaries/build/index.json]: Ooh, yes, you're right :-)

**Pull request closed without merging**